cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/burn_at_relay.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::imports::*;
+use emulated_integration_tests_common::accounts;
+
+/// Create asset and then remove privileges, this result in the burning of the deposit which must
+/// burn at the relay chain.
+#[test]
+fn create_asset_remove_privileges_burn_deposit_at_relay() {
+	let owner = AssetHubWestend::account_id_of(accounts::CHARLIE);
+	let asset_id = 3939;
+
+	// We need first to teleport some balance so that it can be withdrawn later on the relay
+	// chain. Otherwise the asset is not withdrawable due to how genesis is setup.
```

Comment:
```suggestion
	// We need first to teleport some balance so that it can be withdrawn later on the Relay
	// Chain. Otherwise the asset is not withdrawable due to how genesis is set up.
```